appveyor.yml

```diff
@@ -0,0 +1,76 @@
+build: false
+shallow_clone: false
+platform: 'x86'
+clone_folder: C:\projects\work
+branches:
+  only:
+    - appveyor
+  except:
+    - gh-pages
+
+  ## Cache composer bits
+cache:
+  - '%LOCALAPPDATA%\Composer\files -> composer.lock'
+
+services:
+  - mysql
+
+init:
+  #https://github.com/composer/composer/blob/master/appveyor.yml
+  #- SET ANSICON=121x90 (121x90)
+
+# Inspired by https://github.com/Codeception/base/blob/master/appveyor.yml and https://github.com/phpmd/phpmd/blob/master/appveyor.yml
+install:
+  - cinst -y curl
+  - SET PATH=C:\Program Files\curl;%PATH%
+  #which is only needed by the test suite.
+  - cinst -y which
+  - SET PATH=C:\Program Files\which;%PATH%
+  - git clone -q https://github.com/acquia/DevDesktopCommon.git #For tar, cksum, ...
+  - SET PATH=%APPVEYOR_BUILD_FOLDER%/DevDesktopCommon/bintools-win/msys/bin;%PATH%
+  - SET PATH=C:\Program Files\MySql\MySQL Server 5.7\bin\;%PATH%
+  #Install PHP per https://blog.wyrihaximus.net/2016/11/running-php-unit-tests-on-windows-using-appveyor-and-chocolatey/
```

Comment:
ERROR: Running ["C:\windows\System32\wusa.exe"  ... wusa must be enabled. 
Should fix the exit code 1058 failures in appveyor build.
`ps: Set-Service wuauserv -StartupType Manual`

Reply:
Thanks. Added.